docs/releases/1.11.txt

```diff
@@ -226,6 +226,11 @@ Validators
 Backwards incompatible changes in 1.11
 ======================================
 
+:mod:`django.contrib.gis`
+-------------------------
+
+* :ref:`gdalbuild` is now a required dependency for GeoDjango.
```

Comment:
To simplify the codebase and because it's easier to install than when contrib.gis was first released, GDAL is now a required... In older versions, it's only required for SQLite.